app/models/models.scala

```diff
@@ -12,7 +14,7 @@ case class Talk(
   url: String,
   authors: Seq[Author],
   location: String,
-  date: DateTime,
+  date: String,
```

Comment:
Thank you, @philwills ! That was really helpful. I'll open a new PR with the fixes.